src/js/playlist/track.ts

```diff
@@ -1,3 +1,12 @@
+type PlaylistItemTrack = {
+    file?: string;
+    kind: string;
```

Comment:
You can enum or `'valueA' | 'valueB'` the kinds. Will help avoid typos.

`"captions" | "metadata" | "thumbnails" | "chapters"`

I'm not sure if `"subtitles"` is also valid (alias for `"captions"`). I vaguely remember an issue like this but don't see support for it anywhere.

Reply:
:+1: